nuget/lib/dependabot/nuget/native_helpers.rb

```diff
@@ -286,6 +286,22 @@ def self.run_nuget_updater_tool(job_path:, repo_root:, proj_path:, dependency:,
         end
       end
 
+      sig { void }
+      def self.normalize_file_names
+        # environment variables are required and the following will generate an actionable error message if they're not
+        _dependabot_repo_contents_path = ENV.fetch("DEPENDABOT_REPO_CONTENTS_PATH")
+
+        # this environment variable is directly used
+        dependabot_home = ENV.fetch("DEPENDABOT_HOME")
+
+        command = [
+          "pwsh",
+          "#{dependabot_home}/dependabot-updater/bin/normalize-file-names.ps1"
+        ].join(" ")
+        output = SharedHelpers.run_shell_command(command)
+        puts output
```

Comment:
I assume leaving this puts in was intentional in case there's a problem with the shell run?

Reply:
Correct.  The script will report any renames that happened and I want to see them in the log.